doc/plotting.rst

```diff
@@ -702,3 +702,12 @@ You can however decide to infer the cell boundaries and use the
     outside the xarray framework.
 
 .. _cell boundaries: http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#cell-boundaries
+
+One can also make line plots with multidimensional co-ordinates. In this case, ``hue`` must be a dimension name, not a co-ordinate name.
```

Comment:
Nit: I don't think you need an hyphen in coordinates. We write it without hyphen in the [data structures](http://xarray.pydata.org/en/latest/data-structures.html#coordinates) section.